app/_hub/kong-inc/jq/index.md

```diff
@@ -0,0 +1,101 @@
+---
+name: Kong jq
+publisher: Kong Inc.
+versions: 0.1.x
+beta: true
+
+desc: Transform JSON objects included in API requests or responses using jq programs.
+description: |
+  The Kong jq plugin enables arbitrary jq transformations on JSON objects included in API requests or responses.
+  The configuration accepts two sets of options: one for the request and another for the response.
+  For both the request and response, a jq program string can be included, along with some jq option flags
+  and a list of media types. One of the media types must included in the `Content-Type` header for
```

Comment:
```suggestion
  and a list of media types. One of the configured media types must be included in 
  the `Content-Type` header for
```
Not sure if adding "configured" is accurate, but it feels like that should be the case? That is, you're not just including any from a list, you're including one from the _specific_ list that you configured.

Reply:
I think configured is a good modifier here - will update